app/frontend/components/pages/dashboard/dashboardPage.tsx

```diff
@@ -26,11 +26,11 @@ const StakingPage = () => {
     <div className="dashboard desktop">
       <div className="dashboard-column">
         <ShelleyBalances />
-        <CurrentDelegationPage />
+        <DelegationHistoryPage />
```

Comment:
I think a better name for the page would be StakingHistoryPage, since that's what the title of the component also says and it's becoming confusing what is what

Reply:
Fixed, also changed it on other places to make it consistent with itself